server/public/model/incoming_webhook.go

```diff
@@ -52,6 +52,7 @@ type IncomingWebhookRequest struct {
 	Username    string             `json:"username"`
 	IconURL     string             `json:"icon_url"`
 	ChannelName string             `json:"channel"`
+	RootId      string             `json:"root_id"`
```

Comment:
The addition itself is fine, but can you please add a test that covers it? i.e. create a post via webhook and ensure that rootid is correctly set.